tests/framework/helpers/class-wc-helper-reports.php

```diff
@@ -21,5 +21,6 @@ public static function reset_stats_dbs() {
 		$wpdb->query( "DELETE FROM $wpdb->prefix" . \Automattic\WooCommerce\Admin\API\Reports\Products\DataStore::TABLE_NAME ); // @codingStandardsIgnoreLine.
 		$wpdb->query( "DELETE FROM $wpdb->prefix" . \Automattic\WooCommerce\Admin\API\Reports\Coupons\DataStore::TABLE_NAME ); // @codingStandardsIgnoreLine.
 		$wpdb->query( "DELETE FROM $wpdb->prefix" . \Automattic\WooCommerce\Admin\API\Reports\Customers\DataStore::TABLE_NAME ); // @codingStandardsIgnoreLine.
+		\Automattic\WooCommerce\Admin\CategoryLookup::instance()->regenerate();
```

Comment:
This works okay as a workaround for the tests, but I think we should still be adding entries to the lookup table when a new category is created.

When I create a new product category currently, no entry is added until I force a regeneration.

Reply:
The reason I needed to add the regenerate call was that the `uncategorized` category wasn't in the lookup table. When you create a new category it is added to the table. This unit test verifies that https://github.com/woocommerce/woocommerce-admin/blob/update/category-lookup-hierarchy/tests/api/reports-categories.php#L110:L129

Reply:
Ah, that makes sense.

Since `regenerate()` looks like it's already being scheduled, should we just run all scheduled actions here instead of manually calling `regenerate()`?

Reply:
> Since regenerate() looks like it's already being scheduled, should we just run all scheduled actions here instead of manually calling regenerate()?

`regenerate` is scheduled in CRON vs AS. Running the pending actions doesn't generate the table.